docs/examples/cli/README.md

```diff
@@ -0,0 +1,22 @@
+### How to run bysquare in the terminal
+
+Before running bysquare, make sure you have Node.js installed. If you're using `nvm` (Node Version Manager) to manage your Node.js versions, you can set the minimum required Node version for this project with the following command:
+
+```bash
+nvm use
+```
+
+This step is optional, but it ensures that you're using the correct version of Node.js as specified in the .nvmrc file.
+
+To encode data from example.json and print it as a string in the terminal, use the following command:
```

Comment:
Toto sa mi páči, ale dáva to viac zmyslu pri vývoji. Väčšina používateľov knižníc netuší, čo je `nvm` a ako funguje, zároveň nemajú stiahnutý repozitár, aby mohli použiť `nvm use`. Bežne sa používa `npm install -g` alebo `npx`, a v týchto príkladoch by som to neriešil. Toto je skôr pre pokročilých používateľov, ktorí tieto veci ovládajú a rovno sa pozrú do `.sh` súboru. Základné informácie k CLI sú v hlavnom readme.